ss58-registry.json

```diff
@@ -954,6 +945,24 @@
       "decimals": [12],
       "standardAccount": "*25519",
       "website": "https://nftmart.io"
+    },
+    {
+      "prefix": 2207,
```

Comment:
Please order the file based on the prefix.

Reply:
done